data/application.css

```diff
@@ -14,3 +14,15 @@ decoration:backdrop {
         0 3px 6px alpha (#000, 0.23),
         0 14px 28px transparent;
 }
+
+.shortcuts-view {
```

Comment:
These should probably be done using the built-in Gtk margin properties instead of CSS.

Reply:
I see. Will work on that. Is there any technical reason or is just a code design choice?